subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -586,30 +602,24 @@ uint8_t ll_cig_remove(uint8_t cig_id)
 	if (has_cis) {
 		/* Clear configuration only - let CIS disconnection release instance */
 		cig->cis_count = 0;
-	} else {
-		/* No CISes associated with the CIG - release the instance */
-		ll_conn_iso_group_release(cig);
-	}
 
-	return BT_HCI_ERR_SUCCESS;
-}
-
-uint8_t ll_cis_create_check(uint16_t cis_handle, uint16_t acl_handle)
-{
-	struct ll_conn *conn;
-
-	conn = ll_connected_get(acl_handle);
-	if (conn) {
-		struct ll_conn_iso_stream *cis;
+		return BT_HCI_ERR_SUCCESS;
+	}
```

Comment:
There seems to be a hole here... The original code would set the `cis_count` to zero, but I forget how it was supposed to clean up. I think your refactor keeps that problem in, but I need to run some code to be sure. Let's revisit after I have run EBQ with these changes.

Reply:
Also, this for loop might be moved to the above identical loop